dissect/target/helpers/record.py

```diff
@@ -164,8 +164,11 @@ def DynamicDescriptor(types):  # noqa
     "windows/network/interface",
     [
         *COMMON_INTERFACE_ELEMENTS,
-        ("varint", "vlan"),
+        ("string", "vlan"),
+        ("net.IPAddress[]", "subnetmask"),
```

Comment:
Might be nice to use `net.ipnetwork` here instead, and call it `network`, and move it to the common interface elements.